server/routing/lobby.go

```diff
@@ -121,23 +121,24 @@ func (lr *LobbyRoutes) lobbyStartGameHandler(w http.ResponseWriter, r *http.Requ
 		http.Error(w, fmt.Sprintf("no lobby found with ID '%v'", lobbyID), http.StatusNotFound)
 	}
 
+	gameInstance := lr.Services.Game.CreateGame(lobby)
+
 	msg := pogo.GameStartMsg{
 		GameID:  lobby.ID,
 		Players: lobby.Players,
```

Comment:
might want to pull these from the gameInstance you just made. They should be the same, but technically they are more accurate.